src/app/errorPageStyling.tsx

```diff
@@ -0,0 +1,32 @@
+import Paper from "@mui/material/Paper";
+import styled from "styled-components";
+
+export const ErrorCenterer = styled.div`
+    display: flex;
+    justify-content: center;
+    margin-top: 10vh;
+`;
+export const ErrorPanel = styled(Paper)`
+    max-width: 450px;
+    border-radius: 2rem;
+    padding: 5rem;
+    gap: 2rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: ${(props) => props.theme.main.background[0]};
```

Comment:
This is unnecessary - as I think by default in `global_styles.tsx` we have set all paper background and foreground to use `main.background[1]`
For consistency sake with all the papers maybe just use the default color and not set it?
P.S if you still want to use background[0] and override the setting, you should also add `color: ${(props) => props.theme/main/foreground[0]};` in this

Reply:
Removed